src/component/Bookmark.js

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+import BookmarkList from "./BookmarkList";
+
+function Bookmark({ bookmarks, handleChooseRecipe }) {
+  const [showBookmarks, setShowBookmarks] = useState(false);
+
+  return (
+    <div>
+      <button
+        onMouseEnter={() => setShowBookmarks(true)}
```

Comment:
Why are we repeating ourselves here and in `BookmarkList` (violating DRY principles)? Is there a better way to do it? Maybe you should put these in the `<div>` instead?

Reply:
true true